openapi/elasticsearch/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+distribution = "slbeta6"
+license = ["Apache-2.0"]
+keywords = ["Business Intelligence/Analytics", "Cost/Paid"]
+org = "ballerinax"
+name = "elasticsearch"
+icon = "icon.png"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```

Comment:
Shall we go with version 1.0.0 since early connector PRs have used the same version as the initial one. Incase if this is updated please make sure to rebuild & update the Dependencies.toml too.

Reply:
Yeah. will update the versions